src/Lykke.Job.TransactionHandler.Core/Contracts/LimitQueueMessage.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using JetBrains.Annotations;
+using Lykke.Job.TransactionHandler.Core.Domain.Exchange;
+using MessagePack;
+using Newtonsoft.Json;
+
+namespace Lykke.Job.TransactionHandler.Core.Contracts
```

Comment:
Add a description for types which will be shared. The name "contract" implies that it is a shared lib.

Reply:
It is not a shared lib, at least now. It is rabbit message models which are being consumed from queues.